app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,15 @@
+<nav class="navbar navbar-toggleable-xl navbar-inverse bg-inverse">
+  <div class="navbar-nav">
+    <div class="container-fluid">
```

Comment:
In general, container-fluid should be the outer most div in a component/or page

Reply:
Ok